SingularityExecutor/src/main/resources/docker.sh.hbs

```diff
@@ -137,6 +137,19 @@ PARENT_CGROUP=$(get_deepest_cgroup)
 DOCKER_OPTIONS+=(--name=$CONTAINER_NAME)
 DOCKER_OPTIONS+=(--cgroup-parent=$PARENT_CGROUP)
 DOCKER_OPTIONS+=(--env-file=docker.env)
+{{#if runContext.cfsQuota }}
+function get_base_cgroup_directory {
+  if [ -d "/cgroup" ]; then
+    echo "/cgroup"
```

Comment:
These are just mountpoints and people/distros are free to mount the cgroup controllers anywhere. You can get this robustly with findmnt from util-linux (which is installed by default on pretty much every distro).

```
findmnt --kernel --first-only --types cgroup --options cpu --noheadings --output TARGET
```
will output /cgroup/cpu on a HubSpot machine.

Reply:
👍 thanks, will update